ovos_gui/namespace.py

```
@@ -1046,13 +911,13 @@ def _del_namespace_in_remove_timers(self, namespace_name: str):
         if namespace_name in self.remove_namespace_timers:
             del self.remove_namespace_timers[namespace_name]
 
-    def _upload_system_resources(self):
+    def _cache_system_resources(self):
         """
         Copy system GUI resources to the served file path
         """
-        output_path = join(self.gui_file_path, "system")
+        output_path = f"{GUI_CACHE_PATH}/system"
         if exists(output_path):
             LOG.info(f"Removing existing system resources before updating")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)